ui/src/app/workflows/components/workflows-toolbar/workflows-toolbar.tsx

```diff
@@ -10,66 +11,71 @@ require('./workflows-toolbar.scss');
 
 interface WorkflowsToolbarProps {
     selectedWorkflows: Map<string, Workflow>;
-    loadWorkflows: () => void;
-    isDisabled: Actions.OperationDisabled;
+    actionsIsDisabled: Actions.OperationDisabled;
     clearSelection: () => void;
 }
 
-interface WorkflowGroupAction extends WorkflowOperation {
-    groupIsDisabled: boolean;
-    className: string;
-    groupAction: () => Promise<any>;
+interface WorkflowsOperation extends WorkflowOperation {
+    isDisabled: boolean;
+    workflowsAction: () => Promise<any>;
 }
 
-export class WorkflowsToolbar extends React.Component<WorkflowsToolbarProps, {}> {
-    constructor(props: WorkflowsToolbarProps) {
-        super(props);
-    }
+export function WorkflowsToolbar(props: WorkflowsToolbarProps) {
+    const {popup, notifications} = useContext(Context);
+    const numberSelected: number = props.selectedWorkflows.size;
 
-    public render() {
-        return (
-            <Consumer>
-                {ctx => (
-                    <div className={`workflows-toolbar ${this.getNumberSelected() === 0 ? 'hidden' : ''}`}>
-                        <div className='workflows-toolbar__count'>
-                            {this.getNumberSelected() === 0 ? 'No' : this.getNumberSelected()}
-                            &nbsp;workflow{this.getNumberSelected() === 1 ? '' : 's'} selected
-                        </div>
-                        <div className='workflows-toolbar__actions'>{this.renderActions(ctx)}</div>
-                    </div>
-                )}
-            </Consumer>
-        );
-    }
+    const groupAction = useMemo<WorkflowsOperation[]>(() => {
```

Comment:
```suggestion
    const groupActions = useMemo<WorkflowsOperation[]>(() => {
```

this should be plural since there are multiple actions in here

Reply:
or maybe better, we could rename it to `operations` to match the type names
```suggestion
    const operations = useMemo<WorkflowsOperation[]>(() => {
```

Reply:
I think it's better to change the name to 'operations' to match the type name. Actually, I was confused that why the type was `Operation` but the name was `groupAction`.